package-lock.json

```diff
@@ -150,96 +150,64 @@
         "@valtimo/user-interface": "file:dist/valtimo/user-interface",
         "@valtimo/user-management": "file:dist/valtimo/user-management",
         "@valtimo/view-configurator": "file:dist/valtimo/view-configurator",
-        "cypress": "latest",
+        "cypress": "*",
         "deepmerge-ts": "^4.2.2",
         "puppeteer": "^19.8.5",
         "sass": "^1.56.1"
       }
     },
     "dist/valtimo/account": {
       "name": "@valtimo/account",
-      "version": "10.4.0",
+      "version": "0.0.1",
```

Comment:
@ThomasRitense @Hildebrand-Ritense  FYI: Dependabot always seems to change Valtimo dependencies to an invalid version